rmgpy/solver/base.pyx

```diff
@@ -705,8 +707,9 @@ cdef class ReactionSystem(DASx):
         bimolecularThreshold = self.bimolecularThreshold
         trimolecularThreshold = self.trimolecularThreshold
         
-        self.MW = numpy.array([spc.molecularWeight.value_si for spc in coreSpecies+edgeSpecies])
-        self.MWnetwork = numpy.array([sum([sum([x.molecularWeight.value_si for x in r.species]) for r in nwk.reactants]) for nwk in pdepNetworks])
+
```

Comment:
Remove this extra line and can you add another comment to indicate the content of the block?